src/python/ensembl/io/genomio/__init__.py

```diff
@@ -18,4 +18,5 @@
 from .extract_from_gb import *
 from .process_gff3 import *
 from .manifest_maker import *
-from .integrity import *
\ No newline at end of file
+from .integrity import *
```

Comment:
I'm not sure I understand why this __init__.py contains all those imports: what's the use?